pages/AWorkedExample.html

```diff
@@ -44,19 +44,19 @@ <h4>First Card</h4>
   <p>
     Click on <strong>Add Interaction</strong>.
   </p>
-  <img src="static/images/tutorial/card1AddInteraction.png" class="tutorial-image">
+  <img src="static/images/tutorial/card1AddInteraction.png" class="tutorial-image" alt="'Add interaction' button">
 
   <p>
     You'll see the following interaction selector. Go ahead and click on
     <strong>Number Input</strong>.
   </p>
-  <img src="static/images/tutorial/card1NumberInput.png" class="tutorial-image">
+  <img src="static/images/tutorial/card1NumberInput.png" class="tutorial-image" alt="An interaction selector panel with a variety of 'interaction' options given, including the 'number input' option.">
```

Comment:
The use of periods at the end of the alt text is not consistent. Feel free to pick a convention and stick to it.